IOMC/ParticleGuns/interface/FlatProtonLogKsiLogTGun.h

```diff
@@ -0,0 +1,75 @@
+#ifndef IOMC_FlatProtonLogKsiLogTGun_FlatProtonLogKsiLogTGun_h
+#define IOMC_FlatProtonLogKsiLogTGun_FlatProtonLogKsiLogTGun_h
```

Comment:
Change the header accordingly to package directory, i.e. 
`IOMC_FlatProtonLogKsiLogTGun_FlatProtonLogKsiLogTGun_h` -> `IOMC_ParticleGuns_FlatProtonLogKsiLogTGun_h`